### PR TITLE
@trezor/connect: add SLIP 25 support

### PR DESCRIPTION
**Pull request by szymonlesisz**
Based on:
https://github.com/trezor/trezor-suite/pull/6327 and https://github.com/trezor/trezor-suite/pull/6328 and https://github.com/trezor/trezor-suite/pull/6350

Connect:
- Added `TrezorConnect.unlockPath` method
- Use the result of `unlockPath` method to access SLIP-25 keychain (getAddress, getPublicKey, signTransaction)
- Fixed tests for coinjoin using SLIP-25 account
- Fixed `addressPath` in `composeTransaction`

Suite:
- Use SLIP-25 path for coinjoin accounts
- Use `unlockPath` for coinjoin accounts (getAddress, getPublicKey, signTransaction)
- Temporary patch for blockbook responses for SLIP-25 accounts

### CODE REVIEW

File: packages/connect/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # 9.0.2
 
+-   added `unlockPath` method
+-   added `SLIP 25` support
+-   fixed bitcoin extended descriptor (xpubSegwit) for taproot accounts
+
+# 9.0.2
```
Comment (marekrjpolak) [inline packages/connect/CHANGELOG.md:7]: There is `9.0.2` twice.

File: packages/integration-tests/projects/connect/tests/device/authorizeCoinJoin.test.ts [hunk 2]
```diff
@@ -44,66 +50,69 @@ describe('TrezorConnect.authorizeCoinJoin', () => {
...
-            path: ADDRESS_N("m/84'/1'/0'/1/0"),
-            scriptType: 'SPENDWITNESS',
+            path: ADDRESS_N("m/10025'/1'/0'/1'/1/0"),
+            scriptType: 'SPENDTAPROOT',
             userConfirmation: true, // ButtonRequest is not emitted because of preauthorization
             commitmentData,
             preauthorized: true,
         });
 
+        if (!proof.success) throw new Error(proof.payload.error);
```
Comment (marekrjpolak) [inline packages/integration-tests/projects/connect/tests/device/authorizeCoinJoin.test.ts]: Why is this needed right before `expect(proof.success).toBe(true)` ?

**Review by marekrjpolak — DISMISSED** [review]

**Comment by marekrjpolak** [comment]
I forgot to mention that apart from the two comments, everything seems fine and it's working.

File: packages/integration-tests/projects/connect/tests/device/authorizeCoinJoin.test.ts (hunk 2 above)
Comment (szymonlesisz) [reply packages/integration-tests/projects/connect/tests/device/authorizeCoinJoin.test.ts]: right, not needed, thanks

File: packages/connect/CHANGELOG.md (hunk 1 above)
Comment (szymonlesisz) [reply packages/connect/CHANGELOG.md:7]: fixed

File: docs/packages/connect/methods/unlockPath.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+Result is used in `getPublicKey`, `getAddress` and `signTransaction` methods to access requested keychain which is not available by default.
+
+```javascript
+const result = await TrezorConnect.unlockPath(params);
+```
+
+> :warning: **This feature is experimental! Do not use it in production!**
+
+> :note: **Supported only by Trezor T with Firmware 2.4.4 or higher!**
```
Comment (mroz22) [inline docs/packages/connect/methods/unlockPath.md]: maybe 2.5.3 ?

File: packages/connect/src/utils/pathUtils.ts [hunk 4]
```diff
@@ -92,6 +92,7 @@ export const getScriptType = (
         case 84:
             return 'SPENDWITNESS';
         case 86:
+        case 10025:
```
Comment (mroz22) [inline packages/connect/src/utils/pathUtils.ts:93]: nitpick: I would add comment describing what it is (SLIP25)

**Review by mroz22 — APPROVED** [review]
look good, just two nitpicks, feel free to dismiss. 
also sory for the conflicts, should be easy to resolve though

**Comment by szymonlesisz** [comment]
updated, rebased and forcepushed

**Comment by szymonlesisz** [comment]
rebased again, pathUtils tests added [here](https://github.com/trezor/trezor-suite/pull/6325/commits/c8f5ef42947bdd68df18e6bd6553dfe2258c6a5c) didnt reflect recent changes around `script_type` @mroz22

**Review by mroz22 — APPROVED** [review]
